HACKING.rst

```diff
@@ -359,9 +359,9 @@ there may already be differences in tooling that we currently
 work around in less obvious ways.
 
 The high-level plan is to introduce a hierarchy of networking classes
-in ``cloudinit.distros``, which each ``Distro`` subclass will
-reference.  These will capture the differences between networking on
-our various distros, while still allowing easy reuse of code between
+in ``cloudinit.distros.networking``, which each ``Distro`` subclass
+will reference.  These will capture the differences between networking
+on our various distros, while still allowing easy reuse of code between
 distros that share functionality (e.g. most of the Linux networking
 behaviour).  Callers will call ``distro.net.func`` instead of
```

Comment:
distro.networking.func?

Reply:
`distro.net` is where the instantiated `Networking` class is stored (on an instantiated `Distro`). That naming isn't introduced until my first "In more detail" bullet, however, so let me rework this to be clearer.